papers/ReTraCk/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Semantic Parser
-numpy==1.21.0
-torch==1.6.0
+#numpy==1.21.0
```

Comment:
Why comment these out?

Reply:
I use A100 GPU, for which the minimum supported torch version is 1.7.0. 
I put them back, but users may tune them when using different GPUs.